raster-foundry/raster-foundry-api-spec
# Features/lk/update platform org user routes

### Overview
- Update descriptions on platforms/{}/organizations and platforms/{}/members to
reflect permissions required
- Fixed some inconsistent quoting that was causing issues. I defaulted to quoting any text that will eventually be shown to the user (descriptions, summaries, enums)
- Added some missing responses / made them more consistent
- Fixed typos

### Testing/validation
- Verify that swagger editor v2 doesn't complain https://editor2.swagger.io/#!/
- Verify that administration endpoints now show up in the docsite
  To do this locally, you can create a soft link from the docsite's `/assets/spec.yml` to your api spec repository's spec file, and run it locally by using `yarn start`
Closes https://github.com/raster-foundry/raster-foundry-api-spec/issues/23
Closes https://github.com/azavea/raster-foundry-platform/issues/367
